src/DOM.js

```diff
@@ -53,7 +53,13 @@ export default class KenBurnsDOM extends KenBurnsBase<Source> {
       this.container.innerHTML = "";
       this.container.appendChild(source);
       source.style.position = "absolute";
+
       source.style.willChange = "transform"; // already supported by some browsers, ensures GPU acceleration
+      source.style.outline = "1px solid transparent";         // improves edges
+      source.style.backfaceVisibility = 'hidden';             // "
+      source.style['-webkit-backface-visibility'] = 'hidden'; // "
+      source.style.boxShadow = '0 0 1px rgba(255,255,255,0)'; // "
```

Comment:
yeah, it exists in Safari

<img width="437" alt="screen shot 2016-12-22 at 12 41 30" src="https://cloud.githubusercontent.com/assets/211411/21424576/01588004-c844-11e6-8a58-0e85f277ce29.png">